admin/cleanup.py

```diff
@@ -230,22 +251,107 @@ def _filter_test_volumes(self, maximum_age, volumes):
                 keep.append(volume)
         return VolumeActions(destroy=destroy, keep=keep)
 
-    def _destroy_cloud_volumes(self, volumes):
-        """
-        Unconditionally and irrevocably destroy all of the given cloud volumes.
-        """
-        for volume in volumes:
-            try:
-                volume.destroy()
-            except:
-                err(None, "Destroying volume.")
+
+@attributes(['lag', 'prefixes'])
+class CleanAcceptanceNodes(object):
```

Comment:
``PClass`` would be better, but I'm just copying code from Buildbot in this branch, not trying to make improvements.